server/main.ml

```diff
@@ -14,7 +14,7 @@ let info =
     ~man:[`P "This program takes a work directory where every files created \
               are stored. This includes logs, config file and user private \
               keys."]
-    ~version:Config.version
-    Config.name
+    ?version:(Build_info.V1.(Option.map Version.to_string (version ())))
```

Comment:
%{version:pkg} and the version from dune-build-info is not the same value. What's the reason behind this change?

Reply:
My understanding is that [dune-build-info](https://dune.readthedocs.io/en/stable/dune-libs.html#build-info) would provide a nicer version string with the git commit id when opam-health-check installed from git, rather than `0.1~` taken from the opam file. You don't seem interested, so I'll simply remove the commit.